pkg/cli/start.go

```diff
@@ -569,6 +569,28 @@ func setupAndInitializeLoggingAndProfiling(startCtx context.Context) (*stop.Stop
 			" and --log-dir not specified, you may want to specify --log-dir to disambiguate.")
 	}
 
+	if serverInsecure {
+		// Use a non-annotated context here since the annotation just looks funny,
+		// particularly to new users (made worse by it always printing as [n?]).
+		addr := serverConnHost
+		if addr == "" {
+			addr = "<all your IP addresses>"
+		}
+		log.Shout(context.Background(), log.Severity_WARNING,
+			"RUNNING IN INSECURE MODE!\n\n"+
+				"- Your cluster is OPEN for any client that can access "+addr+"!\n"+
+				"- Any user, even root, can log in WITHOUT PROVIDING A PASSWORD!\n"+
+				"- Any user, connecting as root, can READ or WRITE ANY DATA in your cluster!\n"+
+				"- There is NO NETWORK ENCRYPTION, NO TLS, and NO CONFIDENTIALITY!\n\n"+
+				"Check out how to secure your cluster: https://www.cockroachlabs.com/docs/secure-a-cluster.html")
+		go func() {
+			for range time.Tick(5 * time.Minute) {
+				log.Shout(context.Background(), log.Severity_WARNING,
+					"RUNNING IN INSECURE MODE - THE DATA IN THIS CLUSTER IS UNSAFE")
```

Comment:
Yeah, the repeated warning seems really unnecessary. If they didn't see it the first time, repeating it is unlikely to help, and if they did see it then it's just going to be annoying.